README.md

```diff
@@ -30,6 +30,10 @@ handling the ACL and more.
 
 
 ## Supported OSes
+
+Linux kernel v4.13.0 or later is required.
+
+- Debian 9.0 (stretch) - [package status / QA page](https://packages.qa.debian.org/thunderbolt-tools) - [bug reports](https://bugs.debian.org/src:thunderbolt-tools)
```

Comment:
First, thanks for adding it!

Until now, the 'Supported OSes' used to tell which distro we know this to be compiled and installed correctly on (and preferably, testing it on).
Maybe there should be a spearated section for listing links to the relevant pages on distros that include it (like you did here for Debian)?

Reply:
To become an official package in Ubuntu, it has to go through Debian anyway, [even Ubuntu's official documentation says that](https://wiki.ubuntu.com/Upstream).  So when people get it from Ubuntu in future, most of them won't be downloading from Github, they'll be installing the package that has gone through Debian into Ubuntu.

When something is in Debian, any unit tests you include are automatically run each time it is (re)compiled and you can see the build logs, including unit test output, on [the build farm](https://buildd.debian.org/status/package.php?p=thunderbolt-tools&suite=unstable)

Have you had any direct communication with Colin and Kamal, the [maintainers of the Debian package](https://packages.qa.debian.org/t/thunderbolt-tools.html) or any other Debian developers?